.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -3,3 +3,6 @@ contact_links:
   - name: Rust Programming Language Forum
     url: https://users.rust-lang.org
     about: Please ask and answer questions about Rust here.
+  - name: Rust Internals Forum
```

Comment:
I think naming the template "Language feature request" would be more effective at catching the attention. I think the about text is too small to really be noticed

Reply:
Then the other link should probably be renamed as well, since that's what I used as a template